internal/aws/k8s/k8sclient/deployment_info.go

```diff
@@ -0,0 +1,32 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package k8sclient // import "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/k8s/k8sclient"
+
+type DeploymentInfo struct {
+	Name      string
+	Namespace string
+	Spec      *DeploymentSpec
+	Status    *DeploymentStatus
+}
+
+type DeploymentSpec struct {
+	Replicas int32
+}
+
+type DeploymentStatus struct {
+	Replicas            int32
```

Comment:
same question as the Daemonset struct - is a `uint32` more appropriate?

Reply:
Replied above.